README.md

```diff
@@ -36,6 +36,20 @@ appropriate to create a `PersistentVolumeClaim`/`PersistentVolume` and use it in
 Cassandra deployment does not support deleting pods or scaling down, as this might require 
 administrative tasks that are dependent on the final deployment architecture.
 
+### Service Dependencies
+Jaeger production deployment needs an external process to derive dependency links between
+services. Project [spark-dependencies](https://github.com/jaegertracing/spark-dependencies) provides
+this functionality.
+
+This job should be periodically run before end of a day. The following command creates `CronJob`
+scheduled 5 minutes before the midnight.
```

Comment:
I might be getting crazy, but I thought you changed this to run *after* midnight, processing data from the day before?

Reply:
No, it's the current day therefore it runs just before midnight

Reply:
OK, I'm not crazy :-) 

https://github.com/jaegertracing/spark-dependencies/pull/1#discussion_r139058547

I'm not sure it would yield a different result, though, so, I think it's OK to have it the way you did.

Reply:
This is exactly what zipkin uses. If it worked there then it should work for us too. 

If we have a more specific use case we can change it, I just don't want to spend a lot of time on it if it will be replaced by a different thing.

Reply:
That's fine. It's just that it's a weird pattern to process "current day" when "current day" hasn't finished yet.